winthrop/books/management/commands/import_digitaleds.py

```diff
@@ -40,7 +41,7 @@ def import_manifest(self, manifest, path):
 
         # attempt to find the corresponding Winthrop book object
         # for this digital edition and associate them
-        # - edited NYSL winthrop volumes in Plum will have a winthrop
+        # - edited NYSL winthrop volumes in Figgy will have a winthrop
```

Comment:
Need to also update NYSL collection manifest URI on line 69 for figgy equivalent of the old plum collection

Reply:
Nice catch, thank you, @rlskoeser